### PR TITLE
introduce classes for easier CSS selection for custom styling

### PR DESCRIPTION
**Pull request by ryanyun**
## What is this PR for?
This PR attempts to address some of the issues listed below:

1) There are elements in components where the only classes available for CSS selection are hashed through webpack loaders (i.e. `.TabBar__root___15OQR`, `.Comment__goTo___2GeV4`, etc.)
2) Some elements do not have any class names at all
3) Some elements lack wrapping `<div>`s or `<span>`s

By adding the necessary classes to these elements, CSS selection is a lot easier for custom styling.

## What changes have been made?
- "Like" and "Respect" buttons have classes for their respective "Liked" and "Respected" states
- The "Link" button has a wrapping `<span>` tag (_all other buttons had spans around labels_)
- The "OffTopic" and "Featured" tags have clean classes for targeting
- Action Containers in both the "Featured" section and "All Comments" section share class names (for both left and right)
- The "Apply" button in the Configure has a targetable class name
- The "Moderate this stream" link has a targetable class name (and its wrapping container too)
- The "My Profile" and "Configure" sections have class names for respective interior wrappers

### CODE REVIEW

**Comment by okbel** [comment]
Hi! Thank you so much for your contribution! Would you mind checking this PR? https://github.com/coralproject/talk/pull/753

We are currently using classes following that naming convention.

**Comment by ryanyun** [comment]
Hi @okbel!

I have updated the classes to follow the naming convention defined in PR #753 

Although, I left the `.commentActionsLeftContainer` and `.commentActionsRightContainer` as is because it is meant to be shared for `Embed` and non-`Embed` components.

**Comment by kgardnr** [comment]
Hey @ryanyun, you have a syntax error that's causing the build to fail, can you take a look?

Full output here via CircleCI: https://circleci.com/gh/coralproject/talk/5368

```
ERROR in ./plugins/talk-plugin-respect/client/RespectButton.js
Module build failed: SyntaxError: Unexpected token, expected ] (38:57)
````

**Comment by okbel** [comment]
@ryanyun Awesome, thank you! And responding to:
> Although, I left the .commentActionsLeftContainer and .commentActionsRightContainer as is because it is meant to be shared for Embed and non-Embed components.

Hm, no. 🤔  Is there something else using it? If not, feel free to replace it with the correct prefix and casing for it.

**Comment by ryanyun** [comment]
@okbel, the comment actions container `<div>` exists in both the "Featured" and "All Comments" tabs. However, they have unique classes and do not share a class when they function in an almost identical manner.

In "Featured" tab:
image

In "All Comments" tab:
image

Our design team would like to introduce a class (i.e. `.commentActionsLeftContainer` and `.commentActionsRightContainer`) to apply styles to both containers simultaneously without messing with current classes and their respective CSS styling.

Please feel free to suggest better names that follow your naming convention :)

**Comment by kgardnr** [comment]
Hey @ryanyun I think we should still have it prefixed as `talk-embed-stream-` because it's still part of the embedded comment stream. 

And I think we should have them as separate classes for All Comments and Featured Comments, because some newsrooms may be styling them differently. And if you style them the same, it's easy enough to handle this in the stylesheet without duplicating anything.

Thoughts @okbel and @cvle?

**Comment by kgardnr** [comment]
@ryanyun I spoke with @okbel and @cvle, and as long as you add the prefix `talk-embed-stream-` to those 2 classnames, we're good to go and can merge this. Thanks so much!!

**Comment by ryanyun** [comment]
@kgardnr, the two classes have been updated with the prefix `talk-embed-stream-`

Thanks 😄

File: plugins/talk-plugin-like/client/LikeButton.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ class LikeButton extends React.Component {
     return (
       <div className={cn(styles.container, `${plugin}-container`)}>
         <button
-          className={cn(styles.button, {[styles.liked]: alreadyReacted}, `${plugin}-button`)}
+          className={cn(styles.button, {[`${styles.liked} talk-plugin-like-liked`]: alreadyReacted}, `${plugin}-button`)}
```
Comment (okbel) [inline plugins/talk-plugin-like/client/LikeButton.js]: Just a tiny thing here:

There's no need to interpolate here. `cn` is just for that, you can just add ` talk-plugin-like-liked` at the end with ','

**Review by kgardnr — APPROVED** [review]

**Pull request merged**